From 7298aab1a72023ca0658f848b37a498e170f2542 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Mon, 31 May 2021 17:51:16 -0300
Subject: [PATCH] Add missing groups from systemd basic.conf

Add missing groups from systemd sysusers.d/basic.conf to avoid having
them being generated during first boot.

Upstream-Status: Inappropriate [enable feature]

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
Signed-off-by: Jose Quaresma <jose.quaresma@foundries.io>
---
 group.master | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/group.master b/group.master
index d34d2b8..91d8207 100644
--- a/group.master
+++ b/group.master
@@ -36,7 +36,9 @@ plugdev:*:46:
 kvm:*:47:
 staff:*:50:
 games:*:60:
+render:*:61:
 shutdown:*:70:
 wheel:*:80:
+nobody:*:99:
 users:*:100:
 nogroup:*:65534:
-- 
2.34.1

